### ISSUE TITLE
(1.20.1) Serene Seasons foliage color not changing

### ISSUE THREAD

**Issue by NotTohm** [Asek3/Rubidium#664]

**Comment by Knucklesfan** [comment]
Can confirm, this still happens on 1.20.1, with Oculus and Rubidium installed, alongside Biomes O Plenty and a few other QOL mods.

**Comment by PickyPixelStudio** [comment]
Yep, only birch leaves change color, and long grass if Physics mod is installed.
